naming/src/main/java/com/alibaba/nacos/naming/controllers/CatalogController.java

```diff
@@ -150,10 +150,14 @@ public Object listDetail(HttpServletRequest request) {
             List<ServiceDetailInfo> serviceDetailInfoList = new ArrayList<>();
             int pageNo = Integer.parseInt(WebUtils.required(request, "pageNo"));
             int pageSize = Integer.parseInt(WebUtils.required(request, "pageSize"));
-            String keyword = WebUtils.optional(request, "keyword", StringUtils.EMPTY);
+            String serviceName = WebUtils.optional(request, "serviceNameParam", StringUtils.EMPTY);
+            String groupName = WebUtils.optional(request, "groupNameParam", StringUtils.EMPTY);
+            String param = StringUtils.isBlank(serviceName) && StringUtils.isBlank(groupName) ?
+                StringUtils.EMPTY :
+                groupName + Constants.SERVICE_INFO_SPLITER + serviceName;
```

Comment:
maybe here can use NamingUtils.getServiceName() to instead of string concatenation

Reply:
maybe use `NamingUtils.getGroupedName`  😆